packages/ui/src/ui/config/index.ts

```diff
@@ -3,6 +3,20 @@ import _ from 'lodash';
 import UIFactory from '../UIFactory';
 
 import {getConfigData, uiSettings} from './ui-settings';
+import YT from '../config/yt-config';
+import type {UISettings} from '../../shared/ui-settings';
+
+function getClusterSpecificUISettings({
+    uiSettings,
```

Comment:
uiSettings is a constant value from `window.__DATA__`, so it is enough to have one parameter `cluster`.